src/CMake/nativeLnx.cmake

```diff
@@ -64,8 +64,8 @@ INCLUDE (FindBoost)
 # On older systems libboost_system.a is not compiled with -fPIC which leads to
 # link errors when XRT shared objects try to link with it.
 
-# Static linking with Boost is enabled on Ubuntu 18.04.
-if ((${LINUX_FLAVOR} STREQUAL Ubuntu) AND (${LINUX_VERSION} STREQUAL 18.04))
+# Static linking with Boost is enabled on Ubuntu 18.04 and later versions.
+if ((${LINUX_FLAVOR} STREQUAL Ubuntu) AND ((${LINUX_VERSION} STREQUAL 18.04) OR (${LINUX_VERSION} STREQUAL 19.04)))
```

Comment:
GREATER_EQUAL ?

Reply:
I believe we'd want STRGREATER_EQUAL here. I wasn't aware that there is such an operator in CMake, will fix this now.

Reply:
Initially, I would have asked you why didn't you use GREATER_EQUAL instead of STRGREATER_EQUAL since the later comparison is a lexicographically comparison. 

But looking at the conditional line, it looks like you will always be comparing numeric values, so this shouldn't be an issue.